doc/article/en-US/router-configuration.md

```diff
@@ -326,9 +326,8 @@ The function passed to `mapUnknownRoutes()` has to return:
 
         config.title = 'Aurelia';
 
-        let handleUnknownRoutes = (instruction: NavigationInstruction): {moduleId: string} => {
```

Comment:
This isn't wrong, it is just a (necessary?) simplification of the following.

```let handleUnknownRoutes = (instruction: NavigationInstruction): RouteConfig => {```

Though your fix here is also not wrong, it doesn't match the ES2015 version of the code. I would update both places to match, or just fix the typing for the TypeScript version.